src/components/elements/MapPolygonPanel/MapPolygonPanelItem.tsx

```diff
@@ -0,0 +1,168 @@
+import classNames from "classnames";
+import { DetailedHTMLProps, Dispatch, HTMLAttributes, SetStateAction } from "react";
+
+import Text from "@/components/elements/Text/Text";
+import Icon, { IconNames } from "@/components/extensive/Icon/Icon";
+import ModalConfirm from "@/components/extensive/Modal/ModalConfirm";
+import ModalWithLogo from "@/components/extensive/Modal/ModalWithLogo";
+import ModalWithMap from "@/components/extensive/Modal/ModalWithMap";
+import { useModalContext } from "@/context/modal.provider";
+
+import Button from "../Button/Button";
+import Menu from "../Menu/Menu";
+import { MENU_PLACEMENT_RIGHT_BOTTOM } from "../Menu/MenuVariant";
+
+export interface MapPolygonPanelItemProps extends DetailedHTMLProps<HTMLAttributes<HTMLDivElement>, HTMLDivElement> {
+  uuid: string;
+  title: string;
+  subtitle: string;
+  isSelected?: boolean;
+  refContainer?: React.RefObject<HTMLDivElement> | null;
+  setEditPolygon?: Dispatch<SetStateAction<boolean>>;
+}
+
+const MapPolygonPanelItem = ({
+  title,
+  subtitle,
+  isSelected,
+  className,
+  refContainer,
+  setEditPolygon,
+  ...props
+}: MapPolygonPanelItemProps) => {
+  const { openModal, closeModal } = useModalContext();
+  const openFormModalHandlerRequestPolygonSupport = () => {
+    openModal(
+      <ModalWithMap
+        title="Request Support"
+        onClose={closeModal}
+        content="Faja Lobi Project&nbsp;&nbsp;•&nbsp;&nbsp;Priceless Planet Coalition"
```

Comment:
More mocked data, that I'm hoping can be removed?

Reply:
I have removed  file as it contained mocked data that was not being used. Thanks for pointing it out.